.github/workflows/build.yaml

```diff
@@ -11,21 +11,22 @@ jobs:
     runs-on: self-hosted
     timeout-minutes: 20
     container:
-      image: docker.io/mayadata/ms-buildenv:latest
+      image: docker.io/mayadata/ms-buildenv:gila
```

Comment:
Is this the tag that will always be used in future?

Reply:
no, I will swap it out once the PR is approved